Update Readme.md (#10)
DevExpress-Examples/blazor-theme-and-size-mode-switcher: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T845557)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# How to implement a Theme Switcher in Blazor applications

> This example demonstrates how you can add a Theme Switcher to your application. The switcher in this example is the same as in [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/). The choices will include [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).

Reviewer: `The choices will include [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).`

What "choices" are you talking about here? For example, are these two choices something you can select in the "switcher" discussed in the preceding sentence? For example, how about:

`This example demonstrates how you can add a Theme Switcher to your application. The switcher in this example is the same as in the [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/). You can switch between [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).`

Author: ```suggestion
This example demonstrates how you can add a Theme Switcher to your application. The switcher in this example is the same as in the [DevExpress Blazor Demos](https://demos.devexpress.com/blazor/). You can switch between [DevExpress built-in themes](https://docs.devexpress.com/Blazor/401523/common-concepts/customize-appearance/themes) and external Bootstrap themes (the default theme and [free Bootswatch options](https://bootswatch.com/)).
```

Follow the steps below to implement a Theme Switcher in your application:

> 1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application. The newly created project references the **DevExpress.Blazor** NuGet package that contains DevExpress Blazor UI components and links a package with built-in DevExpress themes.

Reviewer 2: Can we name the theme package name in the parentheses?

Author: ```suggestion
1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application configured to use DevExpress Blazor components. The newly created project references the **DevExpress.Blazor.Themes** NuGet package with built-in DevExpress themes (as the **DevExpress.Blazor** package's dependency).

```

2. Copy this example's [switcher-resources](https://github.com/DevExpress-Examples/blazor-theme-switcher/tree/22.2.3%2B/CS/BlazorServer/switcher/switcher/wwwroot/css/switcher-resources) folder to your application's *wwwroot/css* folder. The *switcher-resources* folder has the following structure:

>     * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). Note that the built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package.

Reviewer 2: ...and you do not need to add/link them manually.

Author: ```suggestion
    * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). The built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package that was installed to the application on the previous step.

```

Follow the steps below to implement a Theme Switcher in your application:

1. Use a [DevExpress Project Template](https://docs.devexpress.com/Blazor/401057/get-started) to create a new Blazor Server or Blazor WebAssembly application configured to use DevExpress Blazor components. The newly created project also references the **DevExpress.Blazor.Themes** NuGet package with built-in DevExpress themes.

2. Copy this example's [switcher-resources](https://github.com/DevExpress-Examples/blazor-theme-switcher/tree/22.2.3%2B/CS/BlazorServer/switcher/switcher/wwwroot/css/switcher-resources) folder to your application's *wwwroot/css* folder. The *switcher-resources* folder has the following structure:

>     * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). Note that built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package, which was installed to the application on the previous step.

Reviewer: ```suggestion
    * The *themes* folder includes nested folders whose names correspond to external Bootstrap themes. Each nested folder stores an external theme's stylesheet (the *bootstrap.min.css* file). Note that built-in DevExpress themes are stored separately in the **DevExpress.Blazor.Themes** NuGet package, which was installed to the application in the previous step.
```
